### ISSUE TITLE
Team pings

### ISSUE THREAD

**Issue by nerdman0** [GirafiStudios/Ping#34]
I really like this mod, but one little feature that I want to be added is the ability to create a team and have your ping only show up on your team members screens

**Comment by nerdman0** [comment]
Yooo

**Comment by GirafiStudios** [comment]
Just FYI., it's only implemented in the 1.21.11 version - and going forward obviously.

**Comment by nerdman0** [comment]
yeah, sadly I did see that. I think it does kinda suck since I think for the foreseeable future mods will likely be sticking to 1.21.1 as the new drops system has made keeping modding up to date extremely difficult.
